docs/hyperlinks.md

```diff
@@ -30,7 +30,7 @@ return {
 
     -- file:// URI
     {
-      regex = "\\bfile://\S*\\b",
+      regex = "\\bfile://\\s*\\b",
```

Comment:
This should still be uppercase?

```suggestion
      regex = "\\bfile://\\S*\\b",
```

Reply:
🤔 On nightly that works. I am pretty sure it was pretty unhappy with that on the release version.